components/vocabularies/hpo/annotations/src/test/java/org/phenotips/vocabulary/internal/hpoannotations/NegativePhenotypeForDiseaseAnnotationsExtensionTest.java

```diff
@@ -122,6 +158,17 @@ public void getTargetVocabularyIds() throws Exception
     @Test
     public void getAnnotationSource() throws Exception
     {
+        final Provider<XWikiContext> xcontextProvider = this.mocker2.getInstance(XWikiContext.TYPE_PROVIDER);
```

Comment:
I'm not even sure this test belongs here now, since relocation service should have tests of its own, and this test has nothing to do with `NegativePhenotypeForDiseaseAnnotationsExtension`. So I would just remove it altogether

Reply:
Update: I'm 100% sure this test should be removed. `getAnnotationSource()` method was removed from `AbstractCSVAnnotationsExtension`, so the subject of this test - that method - is no longer here, and no reason for this test to exist.

Reply:
may be it should be relocated :)

Reply:
Actually `getAnnotationSource()` method was not removed completely, it was moved up from `AbstractCSVAnnotationsExtension` to `VocabularyExtension.java` interface. But I will remove all tests if you think they are unnecessary.